### ISSUE TITLE
The performance of PointCNN on TU-Berlin

### ISSUE THREAD

**Issue by jiaxinchenX** [yangyanli/PointCNN#145]
I ran the code of PointCNN on TU-Berlin (using a single Tesla V100 GPU), strictly following the code and hyper-parameter settings on the released github version. The validation accuracy of PointCNN reached 60.% at 40K iteration, but was no longer increased, while the reported result is 70.45% in the NIPS version. I visualized the training/validation loss and training/validation accuracy as follows:
![screenshot from 2019-02-19 09-35-24](https://user-images.githubusercontent.com/32355443/53014915-49701680-3463-11e9-8c91-3ed19581798d.png)
I found that the the loss did not converged to a sufficiently small value. I also tried several small initial learning rates such as 0.001, 0.0001, but got similar results. Do I have missed something that is important to the performance of PointCNN? @burui11087

**Comment by burui11087** [comment]
Hi @cv2drpepper
 
The network performance  you trained on TU-Berlin looks very strange, I need some days to rerun expriment on TU-Berlin to verify the issue you reported. BTW, could you post your computer environments such as CUDA,python,TF version.

Thanks.

**Comment by cv2drpepper** [comment]
Hi @burui11087  

Thanks for your response. My computer environments are: CUDA 9.0; CUDNN 7.3.1; python 3.6 and tensorflow-gpu 1.9.0. 

Thanks

**Comment by burui11087** [comment]
Hi @jiaxin19cvml 

I find that we augment testing datasets when preparing train/val/test datasets so that acc. is just 60%.  I will update code soon in these few days.

Thanks

**Comment by jiaxin19cvml** [comment]
That's fine. Looking forward to your update code. @burui11087

**Comment by jiaxin19cvml** [comment]
That's fine. Looking forward to your update code. @burui11087  Thanks~

**Comment by Leiay** [comment]
Hi, I try to process the test dataset without augmented. Now it has 6666 samples of test. Then I try to reproduce the result you report, still can't get 70.57, but get ~67.
It seems not increase for a long time, is there anything wrong?
Here is the screenshot of tensorboard:

<img width="362" alt="tuBerlin_lr" src="https://user-images.githubusercontent.com/33485544/58598132-d4656780-822f-11e9-9860-a4e739d9926b.png">

<img width="367" alt="tuBerlin" src="https://user-images.githubusercontent.com/33485544/58598129-d0d1e080-822f-11e9-8d0c-474a9903df35.png">
